### PR TITLE
Assign2

### PR DESCRIPTION
**Pull request by blockchainguyy**
documentation of Films API of SWAPI using SWAGGER

### CODE REVIEW

File: assignment2.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    url: "http://swapi.co/api/films/1/"
+schemes:
+- "http"
+paths:
+  /films/:
+    get:
+      tags:
+      - "Films"
+      summary: "get all the film resources"
+      operationId: "get-all-the-films"
```
Comment (devanshutechracers) [inline assignment2.yaml]: what is operationId? Is this required here?

File: assignment2.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        collectionFormat: "multi"
+      responses:
+        200:
+          description: "OK"
+  /films/:id/:
+    get:
+      tags:
+      - "Films"
+      summary: "get a specific film resource"
+      operationId: "get-a-film"
```
Comment (devanshutechracers) [inline assignment2.yaml]: what is operationId? Is this required here?

File: assignment2.yaml [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+      responses:
+        200:
+          description: "OK"
+  /films/schema/:
+    get:
+      tags:
+      - "Films"
+      summary: "view the JSON schema for this resource"
+      description: ""
+      operationId: "get-json-schema"
```
Comment (devanshutechracers) [inline assignment2.yaml]: what is operationId? Is this required here?

**Review by devanshutechracers — CHANGES_REQUESTED** [review]
Remove all uncessary operationId

File: assignment2.yaml [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+                  - ...
+                  "
+      starships:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of starship resource URLs that are in this film."
+        example: -'http://swapi.co/api/starships/2/'
+                 - ...
+
```
Comment (devanshutechracers) [inline assignment2.yaml]: remove unnecessary new lines

File: assignment2.yaml [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+                 - ...
+                 
+      vehicles:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of vehicle resource URLs that are in this film."
+        example: -'http://swapi.co/api/vehicles/4/'
+                 - ...
+
```
Comment (devanshutechracers) [inline assignment2.yaml]: remove unnecessary new lines

File: assignment2.yaml [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+                 - ...
+                 
+      characters:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of people resource URLs that are in this film."
+        example: -'http://swapi.co/api/people/1/'
+                 - ...
+
```
Comment (devanshutechracers) [inline assignment2.yaml]: remove unnecessary new lines

File: assignment2.yaml [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+                 - ...
+                 
+      planets:
+        type: "array"
+        items:
+          type: "string"
+        description: "An array of planet resource URLs that are in this film."
+        example: -'http://swapi.co/api/planets/1/'
+                 - ...
+
```
Comment (devanshutechracers) [inline assignment2.yaml]: remove unnecessary new lines

File: assignment2.yaml [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+        description: "the hypermedia URL of this resource."
+        example: "http://swapi.co/api/films/1/"
+      created:
+        type: "string"
+        description: "the ISO 8601 date format of the time that this resource was created."
+        example: "2014-12-10T14:23:31.880000Z"
+      edited:
+        type: "string"
+        description: "the ISO 8601 date format of the time that this resource was edited."
+        example: "2014-12-12T11:24:39.858000Z"
```
Comment (devanshutechracers) [inline assignment2.yaml:161]: Always add line break at the end of file.

**Review by devanshutechracers — CHANGES_REQUESTED** [review]
Add/Remove Necessary/Unnecessary line breaks.

**Review by devanshutechracers — APPROVED** [review]
Adequate and Precise. Good work.

**Comment by blockchainguyy** [comment]
Thanks for your guidance sir. :)

**Pull request merged**